machine-learning/_posts/2018-12-11-umap.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: review
+title:  "UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction"
+tags:   machine-learning
```

Comment:
May be a new tag named `dimension-reduction` or  `dimensionality-reduction` can be added to the list to better tell these kind of papers?